File: app/src/main/java/com/example/mybusinesstracker/power/PowerActivity.java
package com.example.mybusinesstracker.power;

import android.os.Bundle;

import com.example.mybusinesstracker.BaseCalsses.BaseActivity;
import com.example.mybusinesstracker.R;
import com.example.mybusinesstracker.power.ui.power.PowerFragment;

public class PowerActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //setContentView(R.layout.power_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, PowerFragment.newInstance())
                    .commitNow();
        }
    }
}

File: app/src/main/java/com/example/mohaniceindustry/power/PowerActivity.kt
package com.example.mohaniceindustry.power

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.mohaniceindustry.R
import com.example.mohaniceindustry.power.ui.main.PowerFragment

class PowerActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_power)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                .replace(R.id.container, PowerFragment.newInstance())
                .commitNow()
        }
    }
}